recipes/Components/Langchain_Embeddings_Models.ipynb

```diff
@@ -37,9 +37,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Commented out to avoid downloading the model.\n",
-    "# from langchain_huggingface import HuggingFaceEmbeddings\n",
-    "# embeddings_model = HuggingFaceEmbeddings(model_name=\"all-MiniLM-L6-v2\")"
+    "from langchain_huggingface import HuggingFaceEmbeddings\n",
+    "embeddings_model = HuggingFaceEmbeddings(model_name=\"ibm-granite/granite-embedding-278m-multilingual\")"
```

Comment:
Should this be 30m-english as it is similar in size/language to the embedding it is replacing?